lib/matplotlib/figure.py

```diff
@@ -1484,7 +1484,12 @@ def sca(self, a):
     @docstring.dedent_interpd
     def gca(self, **kwargs):
         """
-        Get the current Axes, creating one if necessary.
+        Get the current Axes.
+
+        If no current Axes exists, a new one is created using `.add_subplot`.
+        (To test whether an Axes exists on a Figure, check whether
+        ``figure.axes`` is empty.  To test whether a current Figure exists,
```

Comment:
```suggestion
        ``figure.axes`` is empty.  To test whether any figures exist,
```
I appreciate that you mean a "current figure" as in a figure on the figure stack, but that technical distinction reads strangely, and I don't think needs to be made here.  Ditto below....

Reply:
I don't have a strong preference, but note that the current wording parallels the first sentence ("if no current Axes exists").  Would you change that too?

Reply:
No, because `gca` stands for "get current axes"....

Reply:
But that's the same as gcf() which stands for `get current figure`?

Reply:
This is not the docs for `gcf`.   You could be more technical and say "To test whether there are any figures in the `gcf()` stack,"  But the phrase "to test whether a current Figure exists" just reads a typo/bad grammar with no context. 

If we want we could change "Get the current Axes from the pyplot Axes stack" to be more explicit as well.  (and maybe replace "stack" by something more correct?)

Reply:
Ah, I think I see the problem.  I pushed a rewording, does that look better.

Reply:
looks good to me.   This could go in regardless of  #20773...